src/Compilers/VisualBasic/Portable/VisualBasicExtensions.vb

```diff
@@ -20,42 +18,44 @@ Namespace Microsoft.CodeAnalysis
 
         ''' <summary>
-        ''' Determines if SyntaxTrivia is a specified kind.
+        ''' Determines if <see cref="SyntaxTrivia"/> is of a specified kind.
         ''' </summary>
-        '''<param name="trivia">The Source SyntaxTrivia.</param>
-        ''' <param name="kind">The SyntaxKind to test for.</param>
+        ''' <param name="trivia">The source trivia.</param>
+        ''' <param name="kind">The syntax kind to test for.</param>
+        ''' <returns><c>true</c> if trivia is of specified kind; <c>false</c> otherwise.</returns>
         <Extension>
         Public Function IsKind(trivia As SyntaxTrivia, kind As SyntaxKind) As Boolean
             Return trivia.RawKind = kind
         End Function
 
         ''' <summary>
-        ''' Determines if SyntaxToken is a specified kind.
+        ''' Determines if <see cref="SyntaxToken"/> is of a specified kind.
         ''' </summary>
-        '''<param name="token">The Source SyntaxToken.</param>
-        ''' <param name="kind">The SyntaxKind to test for.</param>
+        ''' <param name="token">The source token.</param>
+        ''' <param name="kind">The syntax kind to test for.</param>
+        ''' <returns><c>true</c> if token is of specified kind; <c>false</c> otherwise.</returns>
         <Extension>
         Public Function IsKind(token As SyntaxToken, kind As SyntaxKind) As Boolean
             Return token.RawKind = kind
         End Function
 
         ''' <summary>
-        ''' Determines if SyntaxNode is a specified kind.
+        ''' Determines if <see cref="SyntaxNode"/> is of a specified kind.
         ''' </summary>
-        ''' <param name="node">The Source SyntaxNode.</param>
-        ''' <param name="kind">The SyntaxKind to test for.</param>
-        ''' <returns>A boolean value if node is of specified kind; otherwise false.</returns>
+        ''' <param name="node">The Source node.</param>
+        ''' <param name="kind">The syntax kind  to test for.</param>
+        ''' <returns><c>true</c> if node is of specified kind; <c>false</c> otherwise.</returns>
         <Extension>
         Public Function IsKind(node As SyntaxNode, kind As SyntaxKind) As Boolean
             Return node IsNot Nothing AndAlso node.RawKind = kind
         End Function
 
         ''' <summary>
-        ''' Determines if a SyntaxNodeOrToken is a specified kind.
+        ''' Determines if <see cref="SyntaxNodeOrToken"/> is of a specified kind.
         ''' </summary>
-        ''' <param name="nodeOrToken">The source SyntaxNodeOrToken.</param>
-        ''' <param name="kind">The SyntaxKind to test for.</param>
-        ''' <returns>A boolean value if nodeOrToken is of specified kind; otherwise false.</returns>
+        ''' <param name="nodeOrToken">The source node or token.</param>
+        ''' <param name="kind">The syntax kind to test for.</param>
+        ''' <returns><c>true</c> if node or token is of specified kind; <c>false</c> otherwise.</returns>
```

Comment:
```suggestion
        ''' <returns><see langword="true"/> if the node or token is of the specified kind; otherwise, <see langword="false"/>.</returns>
```